src/js/themes/minimal.js

```diff
@@ -0,0 +1,230 @@
+import { window, document } from '../utils/server-safe-globals.js';
+import { MediaThemeElement } from '../media-theme-element.js';
+
+const html = (raw, ...keys) => String.raw({ raw }, ...keys);
+
+const template = document.createElement('template');
+template.innerHTML = html`
+  <style>
+    media-controller {
+      --media-control-background: transparent;
+      --media-control-hover-background: transparent;
+    }
+
+    media-control-bar {
+      height: 42px;
```

Comment:
going by the designs extra small is 30px high and the margins are smaller too

![SCR-20230315-g0v](https://user-images.githubusercontent.com/360826/225377409-7995b960-5f7e-460e-b7a6-04f115c39212.png)